review

Denis-Str opened the pull request.

Denis-Str committed:
review

goodTimeForCoding commented on components/HeaderNavComponent/HeaderNavComponent.vue:
```diff
@@ -88,6 +88,8 @@ import { useZyfraStore } from '@/store/ZyfraStore.js';
 const zyfraStore = useZyfraStore();
 const TABLET_BREAKPOINT = 1024;
+
+// это все похоже на мок данные
```
данные реактивные (показ/скрытие блоков меню при наведении на десктопе, при клике на мобилке)

goodTimeForCoding committed:
Проведёт рефакторинг.

- брейкпоинты привёл к стандартам bootstrap и https://www.notion.so/panfilovdigital/Media-breakpoints-e6af38f001114e6186e88a57f29f9951 идут в направлении от десктопа к мобилке, min-width применён для ограничения контента для широкоформальных экранов
- добавлены безопасные шрифты
- применён @click.self
- добавлен отдельный стор для модалое
- отключён автоимпорт

goodTimeForCoding committed:
Проведёт декомпозицию в компонентах.

goodTimeForCoding committed:
Декомпозиция функции валидации в компоненте формы

goodTimeForCoding committed:
Сгруппирует компоненты и контент в public

goodTimeForCoding committed:
Подключит ESLinnt

goodTimeForCoding committed:
Настроит stylelint.

goodTimeForCoding committed:
Пофиксит ошибки ESLint

goodTimeForCoding committed:
Настроит stylelint для nuxt 3 по регламенту panfilovdigital

goodTimeForCoding committed:
Проведёт рефакторинг

goodTimeForCoding committed:
Настройки в modules вынесет в отдельные файлы
